### PR TITLE
Avoid truncation of REPL output

### PR DESCRIPTION
**Pull request by som-snytt**
Line endings on windows changes the output length.

It was a test issue, @SethTisue, he said ten times testily...

### CODE REVIEW

**Comment by SethTisue** [comment]
is that a Tom Swifty?

**Comment by som-snytt** [comment]
Tom Testily tongue-twister. A Tom Twisty. I'll attempt to forward-port to the PR which hasn't been merged yet. I think I've made this observation before, but the reason you need all the a's is for when you go, "Adriaaaaaan!", adding the echo effect from Khan, of course.

```
scala> $intp.isettings.maxPrintString = 0
             ^
       error: value isettings is not a member of scala.tools.nsc.interpreter.Repl
```

Attempted at https://github.com/scala/scala/pull/6860/commits/9d4d6f5f8192d083521c46f3308c5532d218f128.

File: test/files/run/t10956.scala [hunk 1]
```diff
@@ -10,6 +10,7 @@ import scala.tools.nsc.Settings
 object Test extends ReplTest {
   def code = 
     """
+$intp.isettings.maxPrintString = 0
```
Comment (hrhino) [inline test/files/run/t10956.scala:13]: "don't truncate this", he says, snyppily.

**Comment by SethTisue** [comment]
/me slaps the roof of run 1047: https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-windows/1047/

**Comment by som-snytt** [comment]
```
Sending e-mails to: adriaan@lightbend.com seth.tisue@lightbend.com
ERROR: Could not connect to SMTP host: localhost, port: 25
```
That's OK, Jenkins, they don't reply to email anyway.

**Comment by SethTisue** [comment]
email, what's that?

**Pull request merged**